x-pack/plugins/beats_management/server/rest_api/beats/configuration.ts

```diff
@@ -29,7 +29,7 @@ export const createGetBeatConfigurationRoute = (libs: CMServerLibs) => ({
     try {
       beat = await libs.beats.getById(libs.framework.internalUser, beatId);
       if (beat === null) {
-        return reply({ message: 'Beat not found' }).code(404);
+        return reply({ message: `Beat ${beatId} not found` }).code(404);
```

Comment:
Just a suggestion, I find it can be a better UX to wrap IDs in quotes to show that they're a dynamic part of the error message.

```Beat "my-beat-id" not found```

vs

```Beat my-beat-id not found```